SourceX/DiabloUI/ttf_render_wrapped.cpp

```diff
@@ -40,7 +41,7 @@ SDL_Surface *RenderUTF8_Solid_Wrapped(TTF_Font *font, const char *text, SDL_Colo
 		int w, h;
 		char *spot, *tok, *next_tok, *end;
 		char delim;
-		size_t str_len = SDL_strlen(text);
+		size_t str_len = strlen(text);

```

Comment:
Probably you simply need to include `SDL.h` to be able to use `SDL_strlen` and `SDL_strlcpy`, they are used in other places and appear to be also internally used in SDL it self.

Reply:
The 3ds SDL is buggy with `-std=c++11`, it's fine to use the standard function anyway.